src/main/java/com/powsybl/openloadflow/network/AbstractLfBranch.java

```diff
@@ -88,7 +85,7 @@ protected AbstractLfBranch(LfNetwork network, LfBus bus1, LfBus bus2, PiModel pi
                     sortedLimits.addFirst(LfLimit.createTemporaryLimit(temporaryLimit.getAcceptableDuration(), valuePerUnit));
                 }
             }
-            sortedLimits.addLast(LfLimit.createPermanentLimit(currentLimits.getPermanentLimit() * toPerUnit));
+            sortedLimits.addLast(LfLimit.createPermanentLimit(loadingLimits.getPermanentLimit() * toPerUnit));
```

Comment:
This is not going to work: current limits have a per-unit factor equal to `bus.getNominalV() / PerUnit.SB`, but active power limits and I guess apparent power limits have a per-unit factor only equal to `1 / PerUnit.SB`. Do you have a unit test directly on limits to validate all of that?

Reply:
Yes, in an another place I adapted the scale value used in LimitViolation object building and used getNominalV only in the case of a CURRENT related limit violation. Did not catch this one.
Regarding this function, what you say is simply to adapt toPerUnit variable computation to the real LimitType that's it ? Keep bus.getNominalV() / PerUnit.SB for CURRENT and use 1 / PerUnit.SB for the rest ?